packages/core/scripts/build/generateIcons.ts

```diff
@@ -112,7 +112,7 @@ async function generate(mappings: Record<string, number | undefined> = {}) {
         template: 'css',
         templateClassName: 'icon',
         descent: 200, // Icons were negatively offset on the y-axes, this fixes it
-        templateFontPath: path.relative(stylesheetsDir, fontsDir),
+        templateFontPath: path.relative(stylesheetsDir, fontsDir).replace(/\\/g, '/'),
```

Comment:
There is related logic on line 144. Ideally that would be extracted and de-duplicated, which would help avoid similar bugs in the future.